src/PimpMySql_Query.re

```diff
@@ -64,6 +64,20 @@ let get = (baseQuery, decoder, params, conn) => {
      );
 };
 
+let getWhere = (baseQuery, userQuery, decoder, params, conn) => {
+  let sql =
+    SqlComposer.Select.(
+      {...baseQuery, where: List.concat([userQuery, baseQuery.where])}
+      |> to_sql
+    );
+  let params = PimpMySql_Params.positional(params);
+  log("getWhere", sql, params);
+  Sql.Promise.query(conn, ~sql, ~params?, ())
+  |> Js.Promise.then_(result =>
```

Comment:
Should we make a `PimpMySql.Decode.Promise.rows`? The `.` is the idiomatic way to access sub modules; usually they're re-exported.

Reply:
We could make separate promisified functions for `Decode` in the future.

But for using `.` to only access sub modules, this is not always possible because one of the pain points for having a main index file is circular dependencies.